components/identity-mgt-new/org.wso2.carbon.identity.mgt.admin.ui/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Go through
https://docs.google.com/document/d/1j-OwxDhkCbCuZVvNLXKJShR4R-_swfpotJoAB8-TmI8/edit#heading=h.wz8p6poey31
and clear any obvious issues.